systemtest/fleet_test.go

```diff
@@ -187,7 +187,9 @@ func newAPMIntegrationConfig(t testing.TB, vars, config map[string]interface{})
 			return
 		}
 		t.Logf("elastic-agent logs: %s", output.String())
-		if log, err := agent.APMServerLog(); err == nil {
+		if log, err := agent.APMServerLog(); err != nil {
+			t.Logf("error copying apm-server logs: %s", err)
```

Comment:
Not exactly in the scope of this PR+1 but would be good to log that apm-server logs are not found